valuesets/ValueSet-UKCore-SpecimenBodySite.xml

```diff
@@ -16,15 +16,17 @@
 			<rank value="1" />
 		</telecom>
 	</contact>
-	<description value="A code from the SNOMED Clinical Terminology UK coding system to record a specimen body site."/>
+	<description value="A set of codes to record a specimen body site. Selected from the following SNOMED CT UK coding system:
+&#13; - descendantOf 442083009 | Anatomical or acquired body structure
```

Comment:
Maybe add OR at the end of these, it can of course be found in the filter, but it could also be AND or MINUS here, so if the idea is a quick reference of what is being included, I think adding the operator wouldn't be too messy, but other than that it looks really good.